sdk/src/main/scala/com/horizen/account/AccountSidechainNodeViewHolder.scala

```diff
@@ -96,7 +96,9 @@ class AccountSidechainNodeViewHolder(sidechainSettings: SidechainSettings,
   // since account balances are tracked only in the AccountState.
   // TODO: do we need to notify History with fee payments info?
   override protected def scanBlockWithFeePayments(history: HIS, state: MS, wallet: VL, modToApply: AccountBlock): (HIS, VL) = {
-    (history, wallet.scanPersistent(modToApply))
+    // TODO check this
+    (history, wallet)
+    //(history, wallet.scanPersistent(modToApply))
```

Comment:
I think, `wallet.scanPersistent(modToApply)` is more safe than just `wallet`.
So far we do nothing in `scanPersistent` for Account model, but in case we change anything, we must keep this code in mind.

Reply:
move unsupported operation directly to SidechainWallet, and let AccountWallet return `this`

Reply:
Done